### PR TITLE
info added

### PR DESCRIPTION
**Pull request by Petya-Marinova**
I added my name and birth date

### CODE REVIEW

File: names.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-Ivan Volkov
\ No newline at end of file
+Ivan Volkov
+
+Petya Marinova -04/04
```
Comment (ivanvolkoff) [inline names.txt]: add space after the bar

**Review by ivanvolkoff — CHANGES_REQUESTED** [review]
minor changes requested

File: names.txt (hunk 1 above)
Comment (Petya-Marinova) [reply names.txt]: changes done

**Review by ivanvolkoff — APPROVED** [review]

**Review by yHuKaTa — APPROVED** [review]
OK

**Pull request merged**